File: Examples/Ruby/MathObjects/delete_office_math_object.rb
#ExStart:
require 'aspose_words_cloud'

class WorkingWithMathObjects

  include AsposeWordsCloud
  include AsposeStorageCloud

  def initialize
    #Get App key and App SID from https://cloud.aspose.com
    AsposeApp.app_key_and_sid("APP_KEY", "APP_SID")
    @words_api = WordsApi.new  
  end

  def upload_file(file_name)
    @storage_api = StorageApi.new
    response = @storage_api.put_create(file_name, File.open("../../../data/" << file_name,"r") { |io| io.read } )
  end

  def delete_office_math_object
    file_name = "MathsObject.docx"
    upload_file(file_name)
      
    index = 1
    response = @words_api.delete_office_math_object(file_name, index, opts = {})
  end

end

workingWithMathObjects = WorkingWithMathObjects.new()
puts workingWithMathObjects.delete_office_math_object
#ExEnd:

File: Examples/PHP/MathObjects/DeleteOfficeMathObject.php
//ExStart:
<?php
require_once realpath(__DIR__ . '/..') . '/vendor/autoload.php';
require_once realpath(__DIR__ . '/..') . '/Utils.php';

use Aspose\Words\WordsApi;
use Aspose\Words\AsposeApp;

class MathObjects {

	public $wordsApi;

	public function __construct() {
		AsposeApp::$appSID = Utils::appSID;
		AsposeApp::$apiKey = Utils::apiKey;
		$this->wordsApi = new WordsApi();
	}

    public function deleteOfficeMathObject()
    {       
        // Upload file to Aspose Cloud Storage
        $name = "MathsObject.docx";
        Utils::uploadFile($name);
        
        $index = 1;

        $result = $this->wordsApi->deleteOfficeMathObject($name, $index, $storage=null, $folder=null);
        print_r($result);
    }
}

$mathObjects = new MathObjects();
$mathObjects->deleteOfficeMathObject();

?>
//ExEnd: